Point_set_processing_3/include/CGAL/bilateral_smooth_point_set.h

```diff
@@ -451,7 +451,7 @@ bilateral_smooth_point_set(
     pwns.push_back(Pwn(p, n));
    }
```

Comment:
Indentation issue, here. The issue seems to be line 452, and not in the patch itself.